lib/environment.js

```diff
@@ -7,6 +7,10 @@ const ENV_VARS = {
     process.env.SUBSCRIPTIONS_URL,
     'https://court-backend.eth.aragon.network/subscriptions',
   ],
+  COURT_BACKEND_URL: [
+    process.env.COURT_BACKEND_URL,
+    'https://court-backend.eth.aragon.network',
```

Comment:
I suppose something like:
```
COURT_BACKEND_URL: [
    process.env.COURT_BACKEND_URL,
    `https://court-backend${
      process.env.CHAIN_ID === '4' ? '-rinkeby' : ''
    }.eth.aragon.network`,
  ],
```
should suffice, as if the `CHAIN_ID` is not set, it will be assumed to be 1, therefore not needing the `-rinkeby` interpolation.